core-web/apps/dotcms-ui/.storybook/main.js

```diff
@@ -34,5 +34,6 @@ module.exports = {
         });
 
         return config;
-    }
+    },
+    middleware: './middleware.js'
```

Comment:
Does this means that we need to run a dotCMS in localhost to run the stories?

Reply:
I will try with mocks. Now with Postman and a docker with 23.06, I can see the endpoint shape.